sbt-scalajs-bundler/src/sbt-test/sbt-scalajs-bundler/sharedconfig/common.webpack.config.js

```diff
@@ -0,0 +1,26 @@
+module.exports = {
+  module: {
+    loaders: [
+     {
+        test: /\.(jpe?g|png|gif|svg)$/i,
+        loaders: [
+          'file?hash=sha512&digest=hex&name=[hash].[ext]',
+          'image-webpack?bypassOnDebug&optimizationLevel=7&interlaced=false'
+        ]
+      }
+    ],
```

Comment:
At least the tests passed on the *nix CI. We have so many troubles with Windows support :(